staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -44,6 +44,15 @@ const (
 	// resourceVersionTooHighRetrySeconds is the seconds before a operation should be retried by the client
 	// after receiving a 'too high resource version' error.
 	resourceVersionTooHighRetrySeconds = 1
+
+	// eventRecordDuration is time duration of events we want to keep.
```

Comment:
That actually made me think that we should adjust our bookmark policy to be consistent with it.
I opened https://github.com/kubernetes/kubernetes/issues/90160 for this.

Reply:
address this comment, eventFreshDuration rather than eventRecordDuration